students/mariusz_michalczyk/lesson_08_regex/regex_strip.py

```diff
@@ -1,16 +1,49 @@
 import re
+import unittest
 
 
-def strip_mm(str, replacement=''):
+def strip_mm(text, replacement=''):
     if not replacement:
-        rgx_strip = re.compile(r'\s*(\d*\w*)\s*')
-        matched = rgx_strip.search(str)
+        rgx_strip = re.compile(r'^\s*(\d*\w*)\s*$')
+        matched = rgx_strip.search(text)
```

Comment:
Still `match` would be clearer here, because search is meant to give you multiple results.